src/exceptions.h

```diff
@@ -28,7 +28,12 @@ namespace HeraVM {
 
 class OutOfGasException : public std::exception {
 public:
-  const char* what() const noexcept override { return "Out of gas."; }
+  explicit OutOfGasException(std::string const& _msg = "Out of gas."):
+    msg(std::move(_msg))
```

Comment:
Nope. You cannot move a `const` object (this still will call the copy constructor not the move constructor).
https://www.youtube.com/watch?v=ZKaoR3dP9uM

The canonical way would be:
```
explicit OutOfGasException(std::string _msg = "Out of gas."):
    msg(std::move(_msg))
```

Reply:
It's a typo, there were multiple iterations and added the move at the last minute.

Reply:
Fixed.